README.md

```diff
@@ -42,7 +49,10 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 Create a For Loop that outputs the following string for each piece of fruit in the console. "I want to eat a [fruit]"
 
 ```js
-// write code here
+//
+for(var i = 0; i < foods[0].length; i++) {
+console.log("I want to eat a + foods[0][i]");
```

Comment:
`console.log("I want to eat a " + foods[0][i])` very close!!! just no quotes around foods